RefactoringPracticeI/GameInventory/Inventory/Handlers/SulfurasHandHandler.cs

```diff
@@ -0,0 +1,15 @@
+using RefactoringExerciseI.Enums;
+
+namespace RefactoringExerciseI.Inventory.Handlers
+{
+    class SulfurasHandHandler : ItemUpdateHandler
+    {
+        public override void UpdateItem(Item item)
+        {
+            if (ItemNames.SulfurasHand == item.Name) {
```

Comment:
So if the item is Sulfuras, will return and won't call the next handler?

Reply:
Yes, because at the end of the chain there is logic for what I call "Non special items" and sulfuras is never updated so it is stopped here